src/components/ViewFieldIngelogdExtraInfo/ViewFieldIngelogdExtraInfo.js

```diff
@@ -19,6 +14,15 @@ const eigenarenProperties = [
     'Opdrachtgever',
 ]
 
+/**
+ * Component that renders the ViewFieldIngelogdExtraInfo component and displays a owner list and a button to open the list in the admin area.
+ *
+ * @component
+ *
+ * @param {object} crudObject - Parameter containing a collection of data from the parent state.
+ * @param {boolean} hideEdit - Parameter used to hide the edit field.
+ * @param {string} className - Parameter used to set the style of the div.
```

Comment:
Parameter used to set the style of the div. -> Contains optional extra classNames